mdtraj/geometry/rdf.py

```diff
@@ -109,6 +109,8 @@ def compute_rdf_t(traj, pairs, times, period_length=None, r_range=None,
         Each row gives the indices of two atoms.
     times : array-like, shape=(any, 2), dtype=int
         Each row gives the indices of two frames.
+    num_concurrent_pairs : int, default=100000
```

Comment:
This is a nit but it might be better to name this argument `n_concurrent_pairs` which stays consistent with the naming convention for other arguments (ex: `n_bins`).

Reply:
Got it